Material.Blazor/Components/Dialog/MBDialog.razor

```diff
@@ -1,49 +1,44 @@
 ﻿@namespace Material.Blazor
 @inherits ComponentFoundation
 
+<div @ref="DialogElem"
+     class="mdc-dialog @ActiveConditionalClasses @(@class)"
+     style="@style"
+     id="@id"
+     @attributes="@MyAttributes"
+     aria-modal="true">
 
-@if (IsOpen)
```

Comment:
Most importantly, the dialog is now always rendered. This causes the indentation difference, which makes this look like a huge change